crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -195,6 +195,8 @@ def nodes
       end
 
       def prepare(options = {})
+        ::Crowbar::UpgradeStatus.new.start_step
```

Comment:
It looks like here you are assuming that the upgrade state is `upgrade_prepare`, but how can you be sure?  Same question as https://github.com/crowbar/crowbar-core/pull/736#discussion_r84558612